oracle/oracle-how-to/deploy-oke-nodes-using-ubuntu-images.rst

```diff
@@ -0,0 +1,250 @@
+    --image-source-object-name <object-name> \
+    --bucket-name <bucket-name> \
+    --operating-system "Ubuntu" \
+    --operating-system-version <ubuntu-version-number> \
+    --source-image-type QCOW2 \
+    --wait-for-state AVAILABLE
+
+
+Creating OKE Nodes with Ubuntu Images
+-------------------------------------
+
+The following steps on creating nodes assumes you have an existing OKE cluster on Oracle Cloud, but it is not required to have existing nodes. If you don't have an OKE cluster prepared then Oracle's documentation for `creating a cluster`_ is a good place to start.
+
+Creating Managed OKE nodes with Ubuntu
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Managed nodes are node instances whose lifecycle is managed by the OKE service. Since this is a Limited Availability release of Ubuntu images for OKE, you can only create managed nodes through the Oracle Cloud API (``oci`` CLI or SDK). The ability to create managed from nodes from the Oracle Cloud UI will be added at a later.
+
+To create a managed node, start by copying the following cloud-init script into a file called ``user-data.yaml``.
+
+.. code:: yaml
+
+   #cloud-config
+   
+   runcmd:
+     - oke bootstrap
+
+Then, create a placement configuration file to specify where in Oracle Cloud the Managed node pool should be created and save the file as ``placement-config.json``.
+
+.. code:: json 
+
+   [{
+     "compartmentId":"<compartment-id>",
+     "availabilityDomain":"<availability-domain>",
+     "subnetId":"<subnet-id>"
+   }]
+
+
+Lastly, replace the values and run the following command to create the Managed node pool:
+
+.. code:: bash
+   
+  oci ce node-pool create \
+    --cluster-id=<cluster-id> \
+    --compartment-id=<compartment-id> \
+    --name=<pool-name> \
+    --node-shape=<node-shape> \
+    --size=<pool-count> \
+    --kubernetes-version="1.29.1" \
+    --node-image-id=<ubuntu-image-id> \
+    --placement-configs="$(cat placement-config.json)" \
+    --node-metadata='{"user_data": "'"$(base64 user-data.yaml)"'"}'
+
+
+View the node pool status in Oracle Cloud by navigating to :guilabel:`Kubernetes Clusters (OKE)` and click on your cluster, then select :guilabel:`Resources` > :guilabel:`Node pools` and click on the latest node pool.
+
+Everything will be running as expected when the :guilabel:`Kubernetes node condition` and :guilabel:`Node state` of all the nodes are labelled :guilabel:`Ready`.
+
+Creating Self-Managed OKE nodes with Ubuntu
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following instructions assumes that you have configured your OKE cluster to work with self-managed nodes. If you have not done this, refer to the Oracle documentation for `working with self-managed nodes`_
+
+Before adding a self-managed node, ensure you have configured ``kubectl`` to your OKE cluster with the following command. This process will be easier if ``kubectl`` is configured for a single OKE cluster.
+
+.. code:: bash
+  
+  kubectl cluster-info
```

Comment:
Is that the whole command? Where is the OKE cluster specified? Is that part of what follows in the output of the command? If so, it would be nice to see that too.

Reply:
That's the command as a whole, kubectl defaults to using a config file that we assume the user has set up when we refer to 'configuring kubectl'.

It returns some command returns the cluster status in stdout, would it be worth adding that to the code block?